src/ontology/bridge/uberon-bridge-to-mba.obo

```diff
@@ -966,19 +966,16 @@ relationship: BFO:0000050 UBERON:0001648
 property_value: IAO:0000589 "efferent cochleovestibular bundle" xsd:string
 property_value: status "Verified" xsd:string {source="https://ror.org/03cpe7c52"}
 
-[Term]
-id: MBA:1078
-intersection_of: UBERON:8440077
-intersection_of: BFO:0000050 NCBITaxon:10090
-property_value: IAO:0000589 "rhinal incisure" xsd:string
-property_value: status "Curated" xsd:string {source="https://orcid.org/0000-0001-7258-9596"}
-
 [Term]
 id: MBA:1079
 intersection_of: UBERON:0002985
+intersection_of: UBERON:8440077
+intersection_of: BFO:0000050 NCBITaxon:10090
```

Comment:
I believe this is a regression to the same bug that was spotted in [PR#2695](https://github.com/obophenotype/uberon/pull/2695#issuecomment-1292500068) and fixed in #2701 ...

Reply:
argh, sorry about that, I thought I fixed it in the other repo... I'll look into it. Thanks :)

Reply:
oh I know what happened, copied over verified file which happened after fix, so needed to re-fix it >.< will make the changes